Sources/WalletConnectEcho/EchoClient.swift

```diff
@@ -3,19 +3,20 @@ import WalletConnectNetworking
 
 public class EchoClient {
     private let registerService: EchoRegisterService
-    private let decryptionService: DecryptionService
 
-    init(registerService: EchoRegisterService,
-         decryptionService: DecryptionService) {
+    init(registerService: EchoRegisterService) {
         self.registerService = registerService
-        self.decryptionService = decryptionService
     }
 
     public func register(deviceToken: Data) async throws {
         try await registerService.register(deviceToken: deviceToken)
     }
 
-    public func decryptMessage(topic: String, ciphertext: String) throws -> String {
-        try decryptionService.decryptMessage(topic: topic, ciphertext: ciphertext)
+#if DEBUG
```

Comment:
why we cannot use always `register(deviceToken: Data)` ?

Reply:
it is for simulator id as you get it as a string